### PR TITLE
Modify scripts to use updated aura bribes processor 

### PR DESCRIPTION
**Pull request by shuklaayush**
Addresses #735

### CODE REVIEW

File: scripts/badger/process_bribes_graviaura.py [hunk 1]
```diff
@@ -121,15 +122,35 @@ def sell_weth(badger_total="0"):
...
-            mantissa_sell=aura_share,
-            buy_token=AURA,
-            deadline=DEADLINE,
-            coef=COEF,
-            prod=COW_PROD,
-        )
+        # Check which path is better
+        #  1. Swap WETH for AURA and deposit to GRAVI_AURA
+        #  2. Swap WETH directly for GRAVI_AURA
+        buy_amount_aura = SAFE.cow.get_fee_and_quote(WETH, AURA, aura_share)['buyAmountAfterFee']
```
Comment (petrovska-petro) [inline scripts/badger/process_bribes_graviaura.py:128]: from the api, the come back in str format

```suggestion
        buy_amount_aura = int(SAFE.cow.get_fee_and_quote(WETH, AURA, aura_share)['buyAmountAfterFee'])
```

File: scripts/badger/process_bribes_graviaura.py [hunk 2]
```diff
@@ -121,15 +122,35 @@ def sell_weth(badger_total="0"):
...
+        buy_amount_aura_in_gravi_aura = buy_amount_aura * GRAVI_AURA.totalSupply() // GRAVI_AURA.balance()
+
+        buy_amount_gravi_aura = SAFE.cow.get_fee_and_quote(WETH, GRAVI_AURA, aura_share)['buyAmountAfterFee']
```
Comment (petrovska-petro) [inline scripts/badger/process_bribes_graviaura.py:131]: same as above

```suggestion
        buy_amount_gravi_aura = int(SAFE.cow.get_fee_and_quote(WETH, GRAVI_AURA, aura_share)['buyAmountAfterFee'])
```

File: great_ape_safe/ape_api/cow.py [hunk 3]
```diff
@@ -68,14 +62,30 @@ def _sell(self, sell_token, mantissa_sell, buy_token,
...
+
+        # get the fee and exact amount to buy after fee
+        fee_and_quote = self.get_fee_and_quote(sell_token, buy_token, mantissa_sell)
+
         # grab values needed to post the order to the api
-        fee_amount = int(r.json()['fee']['amount'])
+        fee_amount = int(fee_and_quote['fee']['amount'])
         if mantissa_buy:
             # overwrite quote in case order has a limit
             mantissa_buy == int(mantissa_buy)
```
Comment (gosuto-inzasheru) [inline great_ape_safe/ape_api/cow.py:85]:
```suggestion
            mantissa_buy = int(mantissa_buy)
```
old bug left unfixed :)

File: scripts/badger/process_bribes_graviaura.py [hunk 4]
```diff
@@ -121,15 +122,35 @@ def sell_weth(badger_total="0"):
...
+        else:
+            order_payload, order_uid = SAFE.badger.get_order_for_processor(
+                PROCESSOR,
+                sell_token=WETH,
+                mantissa_sell=aura_share,
+                buy_token=GRAVI_AURA,
+                deadline=DEADLINE,
+                coef=COEF,
+                prod=COW_PROD,
+            )
```
Comment (gosuto-inzasheru) [inline scripts/badger/process_bribes_graviaura.py:152]:
```suggestion
        if buy_amount_aura_in_gravi_aura > buy_amount_gravi_aura:
            buy_token = AURA
        else:
            buy_token = GRAVI_AURA
        order_payload, order_uid = SAFE.badger.get_order_for_processor(
            PROCESSOR,
            sell_token=WETH,
            mantissa_sell=aura_share,
            buy_token=buy_token,
            deadline=DEADLINE,
            coef=COEF,
            prod=COW_PROD,
        )
```

**Review by gosuto-inzasheru — COMMENTED** [review]
tiny code change suggestion but other than that lgtm!

File: scripts/badger/process_bribes_graviaura.py (hunk 4 above)
Comment (shuklaayush) [reply scripts/badger/process_bribes_graviaura.py]: Much better, thanks!

**Review by gosuto-inzasheru — APPROVED** [review]

**Review by petrovska-petro — APPROVED** [review]
great reshape to adapt to new processor ✅

**Pull request merged**